src/inventory/inventory.service.ts

```diff
@@ -34,8 +35,18 @@ export class InventoryService {
     }
   }
 
-  async findAll() {
-    return this.productItemModel.find({});
+  async findAll(args: FindProductItemArgs): Promise<[ProductItem[], number]> {
+    const { first, skip, orderBy } = args;
+    console.log('args', args)
+    const nodes: ProductItem[] = await this.productItemModel
+      .find({})
+      .limit(first)
+      .skip(skip)
```

Comment:
That sounds weird:
> findAll

> limit, skip

That's not what I expect from such a method.
What about `find` instead?

Reply:
I guess that is up to ones preference. For me, `find` involves a query.